widgetssdk/src/main/java/com/glia/widgets/GliaWidgets.java

```diff
@@ -255,6 +257,18 @@ public static void getVisitorInfo(Consumer<GliaVisitorInfo> visitorCallback, Con
         });
     }
 
+    /**
+     * Creates `Authentication` instance for a given JWT token.
+     *
+     * @param behavior authentication behavior
+     * @return {@code Authentication} object or throws {@link GliaException} if error happened.
+     * Exception may have the following cause:
+     * {@link GliaException.Cause#INVALID_INPUT} - when SDK is not initialized
+     */
+    public static Authentication getAuthentication(Authentication.Behavior behavior) {
+        return Dependencies.glia().getAuthentication(behavior);
+    }
```

Comment:
NIT: Just to clarify. What would happen if 'null' is passed to this method?

Reply:
Good question. Thanks for asking. It will behave as if there was no `Behavior`, checks that are made will return `false`, SDK won't crash. But it's better to mark it as `@NotNull`, I made appropriate changes for that now.